dumpling/export/dump.go

```diff
@@ -58,6 +59,17 @@ type Dumper struct {
 	charsetAndDefaultCollationMap map[string]string
 
 	speedRecorder *SpeedRecorder
+	bufChan       chan writeRes
+	wg            *errgroup.Group
+}
+
+func getTable(tables map[databaseName][]*TableInfo) (string, *TableInfo, error) {
+	for db, tbls := range tables {
+		for _, tbl := range tbls {
```

Comment:
nit: check the length of tbls is 1

Reply:
addressed in https://github.com/pingcap/tidb/pull/40553/commits/19ed9f57f3223cf501acbbc9291af45c9b7dcf41